packages/former/src/components/FormRenderer.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <div
+    v-for="node in schema"
+    :key="node._id"
+    class="former-draggable relative"
+    :class="{
+      'bg-zinc-200 rounded': edit === true && showIf ? !showIf(node, nodePath.concat(node._id), data) : false,
+    }"
+    :data-node="node._id"
+  >
+    <component
+      v-if="!edit && showIf ? showIf(node, nodePath.concat(node.name || []), data) : true && errorMessage ? errorMessage(node, data) : true"
```

Comment:
```suggestion
      v-if="!edit && showIf ? showIf(node, nodePath.concat(node.name || []), data) : true"
```

I would add `:error="errorMessage ? errorMessage(node, data) : undefined"` in `FormComponent.vue`  as prop instead as this would allow you to simply use it in the components.